backends-clickhouse/pom.xml

```diff
@@ -27,12 +27,12 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>io.glutenproject</groupId>
+      <groupId>org.apache.gluten</groupId>
```

Comment:
We may need to revert this change, since the custom protobuf-java is still under the old domain name in maven central. Ditto for other pom files where it is introduced.

Reply:
Not just this dependency, but also other ones specified with `custom.protobuf.version`. The custom protobuf has a much larger recursion limit which can avoid an exception for some cases.